pipeline/terraform/modules/fargate_service/variables.tf

```diff
@@ -24,6 +24,18 @@ variable "queue_visibility_timeout_seconds" {
   default = 30
 }
 
+variable "message_retention_seconds" {
+  type = number
+  # The actual default on SQS is four whole days.
+  #   default = 345600
+  # This is sufficient to cope with normal bank holiday weekends.
+
+  # However, if a message is sitting on any main queue for more than a day,
+  # then something has gone rather awry, and it should be moved to a DLQ,
+  # rather than cluttering a main queue.
+  default = 86400
```

Comment:
This will change the behaviour of any queues relying on that default.  However, I don't think 4 days is a reasonable default for main queue retention (even one day is pretty long, we don't have anything that works like that).